app/static/responsive.css

```diff
@@ -35,6 +35,27 @@
     display: none;
   }
 
+  .show-menu {
+    display: block;
+    position: absolute;
+    top: 85px;
+    line-height: 58px;
+    width: 335px;
+    z-index: auto;
+    height: 100%;
+    background-color: #6070ff;
+    margin-top: -85px;
+    text-align: start;
+    opacity: 0.8;
+  }
+
+  .show-menu a {
+    font-size: 800;
+    font-size: 20px;
```

Comment:
- [ ] Please do not use double attributes in your CSS file and try to use the correct attributes provided by the Figma template(remember to import all the font-weights used in the project). Take a look at this:
<img width="418" alt="image" src="https://user-images.githubusercontent.com/61919617/189721543-03357e1c-ab43-44a8-b453-ac1834b8f3d0.png">